packages/react-badge/src/stories/PresenceBadgeStatus.stories.tsx

```diff
@@ -11,6 +11,7 @@ export const Status = () => {
       <PresenceBadge status="doNotDisturb" />
       <PresenceBadge status="offline" />
       <PresenceBadge status="outOfOffice" />
+      <PresenceBadge icon={{ 'aria-label': 'OOF' }} status="outOfOffice" />
```

Comment:
I'd probably move this to another story that is specifically about customizing the icon `aria-label` if we want to show this at all.

Reply:
> if we want to show this at all.

My feeling is that we should not show it here. We're still working out how to consistently handle strings like this for the entire library and this isn't really demonstrating a feature of `PresenceBadge`. It's demonstrating how to use a slot and I think something like this example would be a good addition to the docs section [Customizing Components with Slots](https://61b7c406fa7cd4003a8a07ec-cqsicxkjrr.chromatic.com/?path=/docs/concepts-developer-customizing-components-with-slots--page).

Reply:
yeah, i agree with that. I know sarah has voiced the same concern. There's no reason to have this in any story right now.

Reply:
```suggestion
```